'initial' contribution (#13)
swcarpentry/python-intermediate: _episodes/07-modules.md

---
title: Python Modules
teaching: 50
exercises: 30
questions:

> - "Who can I share my python code for other to easily use?"
> - "How can I create Python code across multiple files?"

Reviewer: ```suggestion
- "How to share my _python_ code for others to use more easily?"
- "How to create _python_ code across multiple files?"
```
